doc/userdoc/styleguide/styleguide.rst

```diff
@@ -0,0 +1,566 @@
+
+.. code-block:: none
+
+ References
+ -----------
+
+ .. [1] Smith J. and Jones M (2009). Title of cool paper. Journal of
+        Awesomeness. 3:7-29. <DOI>
+
+ .. [2] Sander M., et al (2011). Biology of the sauropod dinosaurs: the
+        evolution of gigantism. Biological Reviews. 86(1):117-155.
+        https://doi.org/10.1111/j.1469-185X.2010.00137.x
+
+How we write
+------------
+
+Headings and subheadings
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Headings and subheadings should describe the purpose of the section.
+
+Begin with a descriptive verb or begin with `How to ...`
+
+Headings should explain the section in a short phrase.
+
+Use the verb stem and not the gerund ('ing') form of verbs. Not 'Adding a
+model', but  'Add a model'.
+
+Avoid section names like `Introduction` or `Part 1`.
+
+One word subheadings are acceptable, if the section is short and the meaning is clear.
+
+Use sentence case for headings and subheadings, i.e., begin with an uppercase
+letter but with all other words in lower case (except proper nouns).
+
+
++-----------------------------------+----------------+
+| Good examples:                    | Bad examples:  |
++===================================+================+
+| Create your first neural network  | Start here     |
++-----------------------------------+----------------+
+| How to set up and configure MUSIC | MUSIC and NEST |
++-----------------------------------+----------------+
+| Add a device to your network      | Adding devices |
++-----------------------------------+----------------+
+
+
+Numbers
+~~~~~~~
+
+Numbers should be spelled out if they begin a sentence. In most cases, however,
+the numeral/ordinal format is preferred.
+
+For additional mathematical notation, use the math role or directive.
+
+We use the period for the decimal point. (`57.45`)
+
+The thousand separator is the comma except when showing a code example
+
+   Example:
+
+   We have over 5,000 connections.
+   The number of connections is ``x = 5000``
+
+Make sure you use the correct unit (e.g., millivolts for voltage) and the
+unit's syntax (`V_m`).
+
+Lists
+~~~~~
+
+Use ordered lists for step-by-step instructions only. Do not have more that 2
+related actions in one step.
+
+Use bullet lists to improve clarity of long lists (more than 5 items).
+
+If bullet/ordered list text is a complete sentence, use proper punctuation and
+end with period.
+
+If bullet/ordered list text is an incomplete sentence, do not end with period.
+
+If bullet/ordered list belongs to a sentence, use commas to separate each item
+with the second last time including `and` at end.
+
+Pronouns
+~~~~~~~~
+
+Use the pronouns "you" and "we" whenever possible.
+
+Avoid the pronoun "I".
+
+Avoid gendered terms (NOT police man BUT police officer).
+
+Instead of "guys" or "girls" use inclusive language such as everyone, all,
+members, or folks.
+
+"They" is an acceptable singular third person pronoun
+(see `here <www.merriam-webster.com/dictionary/they>`_).
```

Comment:
I really wished there was a good term to express third-person singular for those who are neither "he" nor "she", without confusing singular and plural. In Norwegian and Swedish and Danish we have "hen" to complement "han" and "hun", but I guess we just got lucky here with an easy starting point. But for now we should follow the developments in American English as you suggest.